SteakBot.Core/Objects/MemeCommand.cs

```diff
@@ -19,5 +20,57 @@ public MemeCommand(MemeResultType resultType, string name, string value, string
             Value = value;
             Description = description;
         }
+
+        #region IEquitable
+
+        public override bool Equals(object other)
+        {
+            var item = other as MemeCommand;
+            return item != null && Equals(item);
+        }
+
+        public bool Equals(MemeCommand other)
+        {
+            if (ReferenceEquals(other, null))
+            {
+                return false;
+            }
+
+            return (object.ReferenceEquals(this.ResultType, other.ResultType)
+                    || this.ResultType.Equals(other.ResultType))
+                && (object.ReferenceEquals(this.Name, other.Name)
+                    || this.Name != null
+                    && this.Name.Equals(other.Name))
+                && (object.ReferenceEquals(this.Value, other.Value)
+                    || this.Value != null
```

Comment:
Use string == operator. It is overloaded and does the same thing as this.Value != null && this.Value.Equal(other.Value),
null check becomes redundand when using == operator on string.